From 21b00908b440894a1e6f537e383f55357299d5c9 Mon Sep 17 00:00:00 2001
From: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
Date: Thu, 29 Oct 2026 15:46:11 +0900
Subject: [PATCH 13/15] packagegroup-renesas-graphics: Fix LICENSE to MIT

LICENSE of a packagegroup is the license of the recipe itself, and the
license of each software in it is declared by its own recipe. CLOSED
was left from the time when gles-user-module was proprietary, and it is
reported as NONE in the SPDX.

Use MIT like packagegroup-renesas-bsp.

Signed-off-by: Yuya Hamamachi <yuya.hamamachi.sx@renesas.com>
---
 recipes-graphics/packagegroups/packagegroup-renesas-graphics.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-graphics/packagegroups/packagegroup-renesas-graphics.bb b/recipes-graphics/packagegroups/packagegroup-renesas-graphics.bb
index 9c36127..a48f177 100644
--- a/recipes-graphics/packagegroups/packagegroup-renesas-graphics.bb
+++ b/recipes-graphics/packagegroups/packagegroup-renesas-graphics.bb
@@ -1,5 +1,5 @@
 SUMMARY = "Renesas package group for Wayland/Weston and OpenGL ES"
-LICENSE = "CLOSED & MIT"
+LICENSE = "MIT"
 
 COMPATIBLE_MACHINE = "(rcar-gen4)"
 PACKAGE_ARCH = "${MACHINE_ARCH}"
